NEWS.md

```diff
@@ -299,6 +299,8 @@ Deprecated or removed
   * Calling `union` with no arguments is deprecated; construct an empty set with an appropriate
     element type using `Set{T}()` instead ([#23144]).
 
+  * `Base.cpad` is removed; use appropriate combination of `rpad` and `lpad` instead ([#23187]).
```

Comment:
Perhaps "is removed" -> "has been removed" or "is deprecated", and perhaps "use appropriate" -> "use an appropriate"? :)